src/utils/tablify.js

```diff
@@ -54,12 +55,17 @@ const reduceKeys = (keys, row) => {
 	return keys.concat(newKeys);
 };
 
-const getKeys = data =>
-	Object.entries(data)
+const getKeys = (data, limit = 3, loop = 1) => {
+	if (loop > limit) {
+		throw new PrintError(
+			'Maximum nesting of the object is exceeded to be displayed as a table.',
```

Comment:
`Output cannot be displayed in table format: Maximum object depth of 3 was exceeded. Consider using JSON output format.`